orange/src/main/java/com/hartwig/hmftools/orange/report/tables/SignatureAllocationTable.java

```diff
@@ -33,13 +33,15 @@ public static Table build(@NotNull String title, float width, @NotNull List<Sign
 
         Cells cells = new Cells(reportResources);
         Table table = Tables.createContent(width,
-                new float[] { 1, 1, 1, 3 },
-                new Cell[] { cells.createHeader("Signature"), cells.createHeader("Allocation"), cells.createHeader("Percent"),
+                new float[] { 1, 2.3F, 1, 1, 1 },
```

Comment:
One note about these numbers: They are all ratios, and I typically don't use floating point numbers.

If you multiple every number by 10 you get the same table (10, 23, 10, 10, 10) which I think would be a bit cleaner (but its a matter of taste!)